.travis.yml

```diff
@@ -10,8 +10,12 @@ go:
 
 go_import_path: github.com/criteo/graphite-remote-adapter
 
+install:
+- go get github.com/golang/lint/golint
+
 script:
 - make style test
+- make lint
```

Comment:
What's the difference with `make style` ?

Reply:
`make style` uses `fmt`: it is only about code formatting.

`go lint` should detect violations from [Effective Go](https://golang.org/doc/effective_go.html) and [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments).